modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/RoutineOutlet.tsx

```diff
@@ -47,7 +47,7 @@ const RoutineOutlet = () => {
 	});
 
 	useEffect(() => {
-		setHeaderActions({actions, item: testrayRoutine, mutate});
+		setHeaderActions({actions: [], item: testrayRoutine, mutate});
```

Comment:
It would be better that you make this validation in a form component.

```suggestion
		setHeaderActions({actions, item: testrayRoutine, mutate});
```